sound/soc/sof/ipc.c

```diff
@@ -216,13 +216,15 @@ static int tx_wait_done(struct snd_sof_ipc *ipc, struct snd_sof_ipc_msg *msg,
 	} else {
 		/* copy the data returned from DSP */
 		ret = msg->reply_error;
-		if (msg->reply_size)
-			memcpy(reply_data, msg->reply_data, msg->reply_size);
 		if (ret < 0)
 			dev_err(sdev->dev, "error: ipc error for 0x%x size %zu\n",
 				hdr->cmd, msg->reply_size);
-		else
+		else {
```

Comment:
we want to be fully compliant with the coding style, so, please also use curly braces in the `if` branch

Reply:
ok